.github/ISSUE_TEMPLATE/new-analysis.md

```diff
@@ -0,0 +1,24 @@
+---
+name: New analysis example
+about: Use this issue template for filing a new analysis issue
+title: 'New Analysis Example:'
+labels: new analysis example
+assignees: ''
+
+---
+
+### What are goals of this new example analysis?
+
+<!-- What should this new example show that's not currently shown in the existing samples -->
```

Comment:
The idea for some new analyses may result from comments on PRs or issues related to existing analyses. Should we also had a section or prompts here on the background for a new analysis issue? Maybe:

```suggestion
<!-- What should this new example show that's not currently shown in the existing samples -->
<!-- What was the background and context that lead to this new example? -->
<!-- Links to github comments or related issues are also helpful --> 
```

Or perhaps we can add a `### Background` header like in the other template in this PR? Not sure which option would be best here as I believe both would get the job done.